src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/ApplicationAppSI.cs

```diff
@@ -14,20 +19,29 @@ namespace Altinn.App.Services.Implementation
     /// </summary>
     public class ApplicationAppSI : IApplication
     {
+        private readonly PlatformSettings _platformSettings;
         private readonly ILogger _logger;
         private readonly HttpClient _client;
 
+        private const string SubscriptionKeyHeaderName = "Ocp-Apim-Subscription-Key";
+
         /// <summary>
         /// Initializes a new instance of the <see cref="ApplicationAppSI"/> class.
         /// </summary>
         /// <param name="logger">the logger</param>
         /// <param name="httpClientAccessor">The Http client accessor </param>
         public ApplicationAppSI(
+           IOptions<PlatformSettings> platformSettings,
             ILogger<ApplicationAppSI> logger,
-            IHttpClientAccessor httpClientAccessor)
```

Comment:
I'm guessing the IHttpClientAccessor both interface and implementation can be deleted if all occurences of it is removed from the SI-classes. 

Also remove the injection from Startup.

Reply:
done